core_backend/validation/validate_retrieval.py

```diff
@@ -104,43 +98,40 @@ def prepare_content_data(
         )
         return df
 
-    def load_content_to_qdrant(
-        self,
-        content_dataframe: pd.DataFrame,
-        vectordb_client: QdrantClient,
+    def load_content_to_db(
+        self, content_dataframe: pd.DataFrame, db_session: pytest.FixtureRequest
     ) -> None:
-        """Load content to qdrant collection"""
+        """Load content to content table"""
         # TODO: Update to use a batch upsert API once created
         n_content = content_dataframe.shape[0]
-        logger.info(f"Loading {n_content} content item to vector DB...")
-        if QDRANT_COLLECTION_NAME not in {
-            collection.name
-            for collection in vectordb_client.get_collections().collections
-        }:
-            create_qdrant_collection(QDRANT_COLLECTION_NAME, QDRANT_VECTOR_SIZE)
+        logger.info(f"Loading {n_content} content item to vector table...")
 
         embedding_results = embedding(
             EMBEDDING_MODEL, input=content_dataframe["content_text"].tolist()
         )
         content_embeddings = [x["embedding"] for x in embedding_results.data]
 
-        points = [
-            PointStruct(
-                id=str(content_id),
-                vector=content_embedding,
-                payload=payload,
+        contents = [
+            ContentDB(
```

Comment:
Wait, I see that we are using ContentDB to add the contents here, but there the text limit is set to 1446. So it doesn't seem like we hit the character limit with the validation dataset?